doc_classes/TrainController.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="TrainController" inherits="Node" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/godotengine/godot/master/doc/class.xsd">
+	<brief_description>
+		Base train controller controlling all train systems
+	</brief_description>
+	<description>
+		Base train controller
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="get_brake" qualifiers="const">
+			<return type="TrainBrake" />
+			<description>
+				Returns TrainBrake class instance
```

Comment:
Returns active train brake.
Nawet nie pisałbym nazwy klasy "TrainBrake", bo może zwrócić jakąś inną klasę pochodną.

Reply:
To jest wygenerowane i nie mam wpływu na to póki nie zmienimy tego directly w kodzie